tt1-blocks/inc/block-styles.php

```diff
@@ -68,7 +68,7 @@ function twenty_twenty_one_register_block_styles() {
 			'core/latest-posts',
 			array(
 				'name'  => 'twentytwentyone-latest-posts-dividers',
```

Comment:
I left these as since the pattern is a Twenty Twenty-One pattern...

Reply:
I left these originally, and I think it's probably ok to keep as is here too. The benefit is that if someone were to switch from the original Twenty Twenty-One to this one, any blocks using those custom styles would still work, since this theme ships with CSS that targets the same classnames.